MC_Setup.cmd

```diff
@@ -85,6 +85,7 @@ put #var MC_IMBUE ROD
 put #var MC_IMBUE.MANA 50
 put #var MC_Enchanting_NOWO 0
 #put #var MC_FOCUS.WAND wood wand
+put #var MC_PREFERRED.ROOM.ARTIF NULL

```

Comment:
Created preferred room for artif. You may have a different preference due to mana.